pkg/sdk/testint/sequences_integration_test.go

```diff
@@ -0,0 +1,167 @@
+package testint
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+ * todo: `ALTER SEQUENCE [ IF EXISTS ] <name> UNSET COMMENT` not works, and error: Syntax error: unexpected 'COMMENT'. (line 39)
```

Comment:
Will you write about this on doc-discuss?

Reply:
done
https://snowflake.slack.com/archives/C6380540P/p1705993709166999

Reply:
Thanks!